lib/preload.ts

```diff
@@ -204,9 +186,18 @@ const createContainer = async (
 	};
 	// Before api 1.25 bind mounts were going to into HostConfig.Binds
 	if (containerOptions.HostConfig !== undefined) {
-		containerOptions.HostConfig[
-			compareVersions(version.ApiVersion, '1.25') >= 0 ? 'Mounts' : 'Binds'
-		] = mounts;
+		if (compareVersions(version.ApiVersion, '1.25') >= 0) {
+			containerOptions.HostConfig.Mounts = mounts.map(([Source, Target]) => ({
+				Source,
+				Target,
+				Type: 'bind',
+				Consistency: 'delegated',
+			}));
+		} else {
+			containerOptions.HostConfig.Binds = mounts.map(
+				([source, target]) => `${source}:${target}`,
+			);
+		}
```

Comment:
This change is to allow the `mounts` variable to be typed, instead of `const mounts: any = [];`.
The previous `bindMount()` function was deleted and the logic moved here because it allows TypeScript to infer and check the type information trough `.map`. (Alternatively it would also have been possible to pass `containerOptions.HostConfig` as an argument to the `bindMount()` function.)